src/util/stringify.ts

```diff
@@ -0,0 +1,80 @@
+type ErrorWithMessage = {
+    message: string;
+    stack?: string;
+};
+
+const isErrorWithMessage = (error: unknown): error is ErrorWithMessage => {
+    return (
+        typeof error === 'object' &&
+        error !== null &&
+        'message' in error &&
+        typeof (error as Record<string, unknown>).message === 'string'
+    );
+};
+
+function stringifyObject(error: object | null): string {
```

Comment:
```suggestion
const stringifyObject = (error: object | null): string => {
```
We should use a consistent style to define methods